tensorflow_probability/python/positive_semidefinite_kernels/rational_quadratic.py

```diff
@@ -165,17 +165,23 @@ def _apply(self, x1, x2, param_expansion_ndims=0):
   @property
   def amplitude(self):
     """Amplitude parameter."""
-    return self.amplitude
+    if "_amplitude" in self.__dict__:
```

Comment:
Can remove these statements and simply return self._amplitude

Reply:
`self._amplitude` is not set in `__init__` if the parameter is None.

Reply:
> Can remove these statements and simply return self._amplitude

`self._amplitude` is not set in `__init__` if the parameter is None.

Reply:
we should always set it, even if it gets set to none.
 as done in other kernels, eg https://github.com/tensorflow/probability/blob/3f7c64714be3fad6cb6f0c25748f44c4cb9b7d3a/tensorflow_probability/python/positive_semidefinite_kernels/exp_sin_squared.py

Reply:
Yep. My above comment (on the __init__ changes) would always set self._amplitude, and so these properties would only have to return the hidden attribute.

Reply:
Yes I got you. Have fixed this.